tests/asserts/keywords.py

```diff
@@ -398,6 +591,17 @@ def process_strats(src, paramData):
                                                             msg = f"🔴ERROR: Destination node not defined:{fromNodeRef}"
                                                             messages["reds"].append(msg)
                                                             messages["counts"]["reds"] += 1
+                                                        if str(fromNode) in roomData["links"]["from"]:
```

Comment:
Does `fromNode` update to become the previous "destinationNode"?

Reply:
I changed this to leverage `search_for_path()` which will try to find a multi-node pathway.
It'll be included in an upcoming commit.